bigbluebutton-html5/imports/ui/components/chat/time-window-list/component.jsx

```diff
@@ -109,6 +109,11 @@ class TimeWindowList extends PureComponent {
     const prevLastTimeWindow = prevTimeWindowsValues[prevTimeWindowsLength - 1];
     const lastTimeWindow = timeWindowsValues[prevTimeWindowsLength - 1];
 
+    if (prevLastTimeWindow?.chatId !== lastTimeWindow?.chatId) {
+      this.systemMessageIndexes = [];
+      this.listRef.forceUpdateGrid();
```

Comment:
in the same componenDidUpdate we've the same logic of force the virtualized list update when changes the [chatId](https://github.com/ramonlsouza/bigbluebutton/blob/develop/bigbluebutton-html5/imports/ui/components/chat/time-window-list/component.jsx#L1316), why did you duplicate it here?

Reply:
you're right, no need to use `forceUpdateGrid` twice, but we need to recompute heights when switching between public and private chat... fixed in c9540df